packages/react-components/react-table/src/components/TableResizeHandle/useTableResizeHandleStyles.styles.ts

```diff
@@ -26,6 +26,17 @@ const useStyles = makeStyles({
     transitionDuration: '.2s',
     zIndex: 1,
 
+    // If the keyboard resize mode was entered with a keyboard, the parent header cell will have
+    // data-fui-focus-within with a border. Displaying a :focus border on the handle then collides with it.
+    // If the keyboard resize mode was entered with a mouse, however, the parent header cell will not have
+    // that data-attribute and we can use the :focus to draw border around the handle, also drawing the attention
+    // of the mouse user to the fact that the handle is interactive and they can drag it with the mouse.
+    ':focus:not([data-fui-focus-visible])': {
```

Comment:
I don't actually see a difference in focus outline behaviour whether I comment out this style or not. Maybe I'm missing something. How can I trigger these focus styles?